lang/c++/cmake/avro-cpp.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
```

Comment:
These are all wrong. Please read the two notes starting at https://github.com/jtojnar/cmake-snips?tab=readme-ov-file#assuming-cmake_install_dir-is-relative-path

Please use the JoinPaths.cmake module there.